search_api_doc.rst

```diff
@@ -0,0 +1,225 @@
+Search API
+==========
+
+Method::
+   
+   GET http://archive.cnx.org/search?q=
+
+Usage::
+   
+   QUERY [author:AUTHOR] [authorID:ID] [title:TITLE] [subject:SUBJECT] ...
+         [keyword:KEY] [type:TYPE] [language:LANG] [pubYear:YEAR] [sort:SORT]
+
+Brakets denote an optional parameter.
+
+
+Parameters:
+
++---------+---------+---------------------------------+
+| Field   | Type    | Description                     |
++---------+---------+---------------------------------+
+| QUERY   | String  | General search string           |
++---------+---------+---------------------------------+
+| AUTHOR  |         | Search by author contact info.  |
++---------+---------+---------------------------------+
+| ID      |         | Search by a specific author ID  |
```

Comment:
So this was something that I was going to bring up for later.   I think what your saying is that the fields column in the table should reflect the case sensitive filter string that we parse for.  For example, our API will accept 'authorID:OpenStaxCollege' but not 'AUTHORID:OpenStaxCollege'.  Is that right?  

If that's the case, you'll notice in the line:
```
QUERY [authorID:ID] 
```
the lower case values such as 'authorID' denote the key needed for the API to interpret the request and the the uppercase values such as 'ID' denote the variables that the user specifies.   I know that its somewhat backwards because we are used to denoting constant values link 'authorID' as fully UPPERCASE strings.  But our API flag strings are set at the moment, so that's why the table field values are all capital.

E.g. If 
```
QUERY = Physics
ID = OpenStaxCollege
```

then 

```
QUERY [authorID:ID]  -->  Physics authorID:OpenStaxCollege
```